File: 180525/codefight/arcade/intro/DoInhan/53-validTime.js
function validTime(time) {
    const hh = time.substr(0, 2);
    const mm = time.substr(time.length-2, 2);
    
    if ((0 <= hh && hh < 24) && (0 <= mm && mm < 60)) {
        return true;
    } else {
        return false;
    }
}

File: 180525/codefight/arcade/intro/JungDongHyun/53. validTime.py
def validTime(time):
    hours = int(time[0:2])
    minutes = int(time[3:6])
    has_valid_hour = hours in range(0, 24)
    has_valid_minute = minutes in range(0, 60)
    return has_valid_hour and has_valid_minute
